package.json

```
@@ -16,7 +16,7 @@
     "format": "pnpm run format:code",
     "format:ci": "pnpm run format:code",
```

Should we also have the format scripts run the import format? e.g. appending a `&& pnpm run format:imports`

I guess we also don't need `format:ci` as it's the same as `format`, and we can update that in `format.yml`

```suggestion
    "format": "pnpm run format:code && pnpm run format:imports",
    "format:ci": "pnpm run format:code && pnpm run format:imports",
```

---

Good call! I didn't think this through enough. Goo catch